helpers/dependency_check.sh

```diff
@@ -260,6 +260,10 @@ dependency_check()
     # linux-exploit-suggester.sh script
     check_dep_file "linux-exploit-suggester.sh script" "$EXT_DIR""/linux-exploit-suggester.sh"
 
+    # readelf
+    export READELF
+    READELF="readelf"
```

Comment:
Why?

Reply:
because of history ;) We have defined it somewhere in the past as variable and so this was the easiest way fixing it again

Reply:
Get rid of the old stuff. 😆

Reply:
Yea I have tried to find a reason of not removing it. ;) As readelf is architecture independent we should go this way ...

Reply:
Thank you